### PR TITLE
Adding use of embed and hyperlinks for ease, of viewing and checking..

### PR DESCRIPTION
**Pull request by BusyLeon**
• Added use of discord embeds, so future (more expansive) modifications can be made in the future.. 
without use of embeds a lot of features within discord web-hooks and bots cannot be used.. In which we have reached some point where such features.. that embeds allow would be very viewing friendly!

• Added hyperlink, linking to the Catalogs of which are listed!

Up to my testing with both, discord bots and discord web-hooks, this shall work accordingly..

### CODE REVIEW

File: src/discord-update-report.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ function formatReport(report) {
   out.push('');
   out.push('**CATALOG UPDATE**');
   for (const r of report) {
-    out.push(`- ${r.catalog} :`);
+    out.push(`- [${r.catalog}](https://keycap-archivist.com/maker/${r.catalog.toLowerCase().replace(/\s+/g, "-")}) :`);
```
Comment (zekth) [inline src/discord-update-report.js:9]: Not sure if we'll have difference with the makers name but if you want to be tight to the generated slugs in website please refer to: https://github.com/keycap-archivist/website/blob/91a3565a29ea034de227dd50dc74b950e4ec9825/gatsby-node.js#L16

**Review by zekth — COMMENTED** [review]
Minor comment but not sure it will lead to a bug.

Also, will all the links lead to embedded previews or this will not happen?

File: src/discord-update-report.js (hunk 1 above)
Comment (BusyLeon) [reply src/discord-update-report.js:9]: If you mean embedded previews as in when you link a website and discord shows the embed meta from the site... then no, by default.. Discord hyperlinks do not display embeds beneath them.. As if they did, this pull request ( if approved ) would make catalog updates look horrendous.. and you'd have to scroll up to see the majority of the catalog changes..

Comment (BusyLeon) [reply src/discord-update-report.js:9]: I'll see about the slugify module.. it seems that'll be helpful..

**Comment by zekth** [comment]
@BusyLeon you need to add the package with `yarn` and commit the `package.json` and `yarn.lock`

**Comment by BusyLeon** [comment]
That should be correct!

**Review by zekth — APPROVED** [review]

**Comment by zekth** [comment]
Thanks!

**Comment by zekth** [comment]
@BusyLeon https://github.com/keycap-archivist/database/runs/2821788275?check_suite_focus=true#step:11:28
Does not work

**Pull request merged**